dsc_lib/src/configure/config_doc.rs

```diff
@@ -134,9 +134,13 @@ pub struct Resource {
     #[schemars(regex(pattern = r"^\[resourceId\(\s*'[a-zA-Z0-9\.]+/[a-zA-Z0-9]+'\s*,\s*'[a-zA-Z0-9 ]+'\s*\)]$"))]
     pub depends_on: Option<Vec<String>>,
     #[serde(skip_serializing_if = "Option::is_none")]
+    pub kind: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
     pub properties: Option<Map<String, Value>>,
     #[serde(skip_serializing_if = "Option::is_none")]
     pub metadata: Option<Map<String, Value>>,
+    #[serde(rename = "securityContext", skip_serializing_if = "Option::is_none")]
+    pub security_context: Option<String>,
```

Comment:
`securityContext` is part of metadata already

Reply:
Is the goal to surface `securityContext` in the resource metadata or the resource's Microsoft.DSC metadata?

Reply:
It should be within the `Microsoft.DSC` metadata where we have `securityContext` already